TypeCobol/Compiler/Parser/CodeElementsParserStep.cs

```diff
@@ -344,11 +342,36 @@ internal static IList<DocumentChange<ICodeElementsLine>> ParseProcessedTokensLin
                                 }
                             }
 
+                            //Report diagnostics if some incomplete CE have been encountered previously
+                            if (diagnosticsToReport.Count > 0)
+                            {
+                                if (codeElement.Diagnostics == null)
+                                {
+                                    codeElement.Diagnostics = new List<Diagnostic>(diagnosticsToReport);
+                                }
+                                else
+                                {
+                                    foreach (var diagnostic in diagnosticsToReport)
+                                    {
+                                        codeElement.Diagnostics.Add(diagnostic);
```

Comment:
Optional suggestion:
Maybe change `codeElement.Diagnostics` to List and use `AddRange` here.
I don't think we'll ever use another implementation of IList for `CodeElement.Diagnostics`.

Reply:
done